test/sqllogictest/transform/column_knowledge.slt

```diff
@@ -238,34 +236,34 @@ Explained Query:
   Return // { arity: 1 }
     CrossJoin type=differential // { arity: 1 }
       implementation
-        %0:l1[×]Uef » %1[×]Uef
```

Comment:
This looks like a possible regression, losing the `U` symbol. It references a `Union` pattern that looks like it could/should have unique keys, but perhaps it does not.